pyop2/types/dataset.py

```diff
@@ -330,32 +335,22 @@ def dm(self):
 
 class MixedDataSet(DataSet):
     r"""A container for a bag of :class:`DataSet`\s.
-
     Initialized either from a :class:`MixedSet` and an iterable or iterator of
     ``dims`` of corresponding length ::
-
```

Comment:
Why have these been modified? I think the blank lines here aid readability to distinguish the code blocks from the rest of the text.

Reply:
Seems like my editor just deleted them. I will revert it.